build.sbt

```diff
@@ -3,14 +3,13 @@ import de.heikoseeberger.sbtheader.HeaderPattern
 
 name := "akka-stream-kafka"
 
-val akkaVersion = "2.4.17"
+val akkaVersion = "2.5.0"
```

Comment:
@patriknw So if we change 2.5.0 back to 2.4.17 in this PR then this version will be working well with 2.5.0, right?